src/poliastro/czml/extract_czml.py

```diff
@@ -29,8 +29,8 @@ def __init__(self, start_epoch, end_epoch, N):
             of sampled data points will be N when calling
             add_orbit()
         """
-        self.czml = dict()
-        self.orbits = []
+        self.czml = dict()  # type: Dict[int, Any]
```

Comment:
`from typing import Dict, List, Any`

Reply:
Thank you! I was really confused as to why it was throwing this error...